addons/misra.py

```diff
@@ -2691,4 +2702,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    main()
```

Comment:
(Cosmetic issue: [it is considered](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) a good practice to ends files with a new line. It would also make the diff a bit smaller!)

Reply:
Thanks, it modified in next commit.